scheduler/src/cook/mesos/api.clj

```diff
@@ -1111,8 +1112,9 @@
         ;; if the query params are coerceable, both :job and "job" keys will be
         ;; present, but in that case we only want to read the coerced version
         query-jobs (or query-jobs-coerced (mapv #(UUID/fromString %) query-jobs-uncoerced))
+        body-job (vectorize (get-in ctx [:request :body-params :job]))
```

Comment:
Hmm.  As far as I'm aware it hasn't yet been needed in another namespace.  That is not the only barometer for a function should be moved to a util namespace, but in general, in my opinion, treating a piece of data as "could be either an array or a single value" is not something that we'd want to encourage more of (at least not as a first option) - it's a sign of weak schema (or stretching for backwards compatibility as is being done here).